psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -2411,16 +2418,18 @@ void RDFMP2::form_Z()
         Dtemp->scale(0.5);
 
         // Add in the reference contribution
+
         for (int i = 0; i < nocc; ++i)
             Dtemp->add(i, i, 1.0);
-        Ca_ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
-        epsilon_a_ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
+        Ca__ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
+        epsilon_a__ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
```

Comment:
You are right, I was thinking of including the SCF Wavefunction data for post-scf methods which  i guess can be done quite easily from the python's side:
```
scf_e, scf_wfn = energy('scf', return_wfn=True)
scf_fchk_writer = core.FCHKWriter(scf_wfn)
scf_fchk_writer.write('scf.fchk')
mp2_e, mp2_wfn = gradient('mp2', ref_wfn = scf_wfn, return_wfn=True)
mp2_fchk_writer = core.FCHKWriter(mp2_wfn)
mp2_fchk_writer.write('mp2.fchk')
```
Also, do you think we should change the name `Alpha Orbital Energies` which actually are the occupation numbers in the case of df-mp2 to something more suitable?